problem_builder/public/js/step.js

```diff
@@ -110,20 +105,15 @@ function MentoringStepBlock(runtime, element) {
             return $('.sb-step', element).data('has-question');
         },
 
-        updateChildren: function() {
-            children.forEach(function(child) {
-                var type = $(child.element).data('block-type');
-                switch (type) {
-                case 'video':
-                    updateVideo(child);
-                    break;
-                case 'sb-plot':
-                    updatePlot(child);
-                    break;
-                }
-            });
-        }
+        show: function () {
+            $(element).show();
+            childManager.show();
```

Comment:
@jbzdak Based on the way `show` and `hide` are used here, it might make sense to call them `showAll` and `hideAll` instead.